resources/qml/WelcomePages/WhatsNewContent.qml

```diff
@@ -21,39 +25,170 @@ Item
         anchors.top: parent.top
         anchors.horizontalCenter: parent.horizontalCenter
         horizontalAlignment: Text.AlignHCenter
-        text: catalog.i18nc("@label", "What's new in Ultimaker Cura")
+        text: catalog.i18nc("@label", "What's New")
         color: UM.Theme.getColor("primary_button")
         font: UM.Theme.getFont("huge")
         renderType: Text.NativeRendering
     }
 
-    Cura.ScrollableTextArea
+    Item
     {
-        id: whatsNewTextArea
-
+        id: topSpacer
         anchors.top: titleLabel.bottom
-        anchors.bottom: getStartedButton.top
-        anchors.topMargin: UM.Theme.getSize("wide_margin").height
-        anchors.bottomMargin: UM.Theme.getSize("wide_margin").height
-        anchors.left: parent.left
-        anchors.right: parent.right
+        height: UM.Theme.getSize("default_margin").height
+        width: UM.Theme.getSize("default_margin").width
+    }
+
+    Rectangle
+    {
+        anchors
+        {
+            top: topSpacer.bottom
+            bottom: whatsNewDots.top
+            horizontalCenter: parent.horizontalCenter
+            margins: UM.Theme.getSize("default_margin").width
+        }
+
+        height: parent.height - (UM.Theme.getSize("default_margin").height * 2)
+        width: parent.width - (UM.Theme.getSize("default_margin").width * 2)
+        color: UM.Theme.getColor("viewport_overlay")
+
+        StackLayout
+        {
+            id: whatsNewViewport
+
+            anchors
+            {
+                top: parent.top
+                horizontalCenter: parent.horizontalCenter
+                margins: UM.Theme.getSize("default_margin").width / 2
```

Comment:
Right now this means that you only have a topMargin (since top is the only anchor that you defined). Is there a specific reason that you did that?

Reply:
Yes, the text(field) seems to go on to the bottom of the slightly-more-grey rectangle in the designs. I found out that this is for a good reason, because if the text ends up large enough to need scrolling, a grey margin without border looks really weird. I'll make it into topMargin to prevent everyone reading this code from having the same question though :-p